src/main/resources/api/service_openapi.yaml

```diff
@@ -24,6 +24,37 @@ paths:
           description: Service is broken
           $ref: '#/components/responses/StatusResponse'
 
+
+  '/api/janitor/v1/resource/{type}':
+    parameters:
+      - $ref: '#/components/parameters/Type'
+    post:
+      description: |
+        Create a new tracked resource.
+      operationId: createResource
+      tags:
+        - janitor
+      requestBody:
+        description: |
+          Resource UID, time to live, resource labels
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreateResourceRequestBody'
+      responses:
+        200:
+          description: Ok
+          $ref: '#/components/responses/CreateResponse'
+        400:
+          description: Bad request - invalid id, badly formed
+          $ref: '#/components/responses/ErrorResponse'
+        401:
+          description: Permission denied
+          $ref: '#/components/responses/ErrorResponse'
+        500:
+          description: Creation request error
+          $ref: '#/components/responses/ErrorResponse'
+
   '/api/janitor/v1/resource/{id}':
```

Comment:
maybe add a `// TODO remove temporrary placeholder endpoint.` to make it clear what's permanent vs temporary.